compiler/test/suites/imports.re

```diff
@@ -204,18 +150,18 @@ describe("imports", ({test, testSkip}) => {
   /* Misc import tests */
   assertCompileError(
     "test_bad_import",
-    "{let x = (1, 2); import * from \"tlists\"; x}",
+    "{let x = (1, 2); include \"tlists\" as TLists; x}",
     "error",
   );
   assertFileRun("test_file_same_name", "list", "OK\n");
   assertSnapshot(
     "annotation_across_import",
-    "import TList, { Empty } from \"tlists\"; let foo : TList.TList<String> = Empty; foo",
+    "include \"tlists\" as TLists; from TLists use { type TList }; let foo : TLists.TList<String> = Empty; foo",
```

Comment:
Why "use" the type but then still access it from the module itself?

Reply:
This test is testing that even though you used `Empty` from the `use`d type, your annotation from the type on the module still typechecks.